Mathlib/Analysis/SpecialFunctions/Pow/Asymptotics.lean

```diff
@@ -51,6 +51,76 @@ theorem tendsto_rpow_neg_atTop {y : ℝ} (hy : 0 < y) : Tendsto (fun x : ℝ =>
     (tendsto_rpow_atTop hy).inv_tendsto_atTop
 #align tendsto_rpow_neg_at_top tendsto_rpow_neg_atTop
 
+open Asymptotics in
+lemma tendsto_rpow_atTop_of_base_lt_one (b : ℝ) (hb₀ : -1 < b) (hb₁ : b < 1) :
+    Tendsto (rpow b) atTop (𝓝 (0:ℝ)) := by
+  show Tendsto (fun z => b^z) atTop (nhds 0)
+  rcases le_or_gt b 0 with hb | hb
```

Comment:
You should be able so streamline this with `rcases lt_trichotomy b 0 with hb|rfl|hb` which does directly the three cases distinction.